tools/ci_testing/addons_cpu.sh

```diff
@@ -42,9 +44,9 @@ export TF_NEED_CUDA=0
 
 # Check if python3 is available. On Windows VM it is not.
 if [ -x "$(command -v python3)" ]; then
-    python3 ./configure.py
+    python3 ./configure.py $1
   else
-    python ./configure.py
```

Comment:
Would you mind adding a check of the passed arg and return error if it's not `no-deps`

Reply:
Sure thing

Reply:
This file would actually deserve to be a python file. Maybe I'll open a pr to convert it in the future.